backend/services/src/shared/casl/policy.guard.ts

```diff
@@ -8,7 +14,9 @@ import { Action } from "./action.enum";
 import { CaslAbilityFactory, AppAbility } from "./casl-ability.factory";
 import { CHECK_POLICIES_KEY } from "./policy.decorator";
 import { PolicyHandler } from "./policy.handler";
-const { rulesToQuery } = require('@casl/ability/extra');
+const { rulesToQuery } = require("@casl/ability/extra");
+
+const unAuthErrorMessage = "This action is unauthorised";
```

Comment:
Don't we need to add this to localisation file?

Reply:
@palindaa the scenarios we are getting this error is while only sending the req via swagger or postman right? so I didn't add that, I'll check again anyway I will add this to the localisation file